modules/core/src/main/java/org/apache/ignite/IgniteCluster.java

```diff
@@ -453,6 +453,21 @@
      */
     public void active(boolean active);
 
+    /**
+     * Checks Ignite grid in read-only mode or not.
```

Comment:
Javadoc is not very clear. Consider consulting someone english speaking to improve doc quality in public interfaces.